### ISSUE TITLE
add edit link to packages / channels / streams

### ISSUE THREAD

**Issue by fjenett** [PieceMeta/piecemeta-angular-frontend#5]
So it's easier to get there and one does not have to go through the full path of package --> channel --> stream every time …

**Comment by dasantonym** [comment]
not exactly sure where what should go where here. so you want an edit link to streams directly from the edit package page (full listing then, would load all the streams...)? the items listed on the edit pages already link to their respective edit page.

**Comment by fjenett** [comment]
No, form the "show xxx" pages … so viewing content you can go edit it directly instead of having to go through the whole chain of edit pages again.
